src/sentry/notifications/notifications/strategies/role_based_recipient_strategy.py

```diff
@@ -51,15 +53,13 @@ def determine_recipients(
         # convert members to users
         return user_service.get_many(filter={"user_ids": [member.user_id for member in members]})
 
-    def determine_member_recipients(self) -> Iterable[OrganizationMember]:
+    def determine_member_recipients(self) -> QuerySet[OrganizationMember]:
```

Comment:
Will changing the return type here affect anything else?

Reply:
no, a queryset is also iterable

Reply:
but i was forcibly casting it to an iterable before